utils/printTracks.py

```diff
@@ -58,4 +76,4 @@ def printTracks(tracksDir, filename=None):
 
     tracksDir = "../tracks"
 
-    printTracks(tracksDir, filename=None)
+    printTracks(tracksDir, filename='tracks.csv')
```

Comment:
please leave to None since we don't want to actively change the track.csv file every time